app/com/arpnetworking/metrics/portal/reports/EmailBuilder.java

```diff
@@ -0,0 +1,36 @@
+package com.arpnetworking.metrics.portal.reports;
+
+import java.util.Properties;
+
+import javax.mail.BodyPart;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+
+
+public class EmailBuilder {
```

Comment:
A builder typically refers to a type of class that constructs instances of another class. While it obviously can have other meanings, I would prefer to avoid that word here, especially given how heavily this codebase uses the Builder design pattern.